src/statics.rs

```diff
@@ -61,13 +69,100 @@ pub(crate) static EVENT_METADATA: once_cell::sync::Lazy<
             next_pos += 1;
         }
 
-        let map = dashmap::DashMap::with_capacity(events_slice.len());
+        let bh = FnvHasher::default();
+
+        let mut map: Box<[core::mem::MaybeUninit<crate::_details::ParsedEventMetadata>]> = Box::new_uninit_slice(good_pos + 1);

```

Comment:
Clippy says new_uninit_slice was only stabilized last month (1.82), so maybe I should hold off on using it. Though github's CI is apparently running 1.82 already and so are the internal pipelines, so it's probably fine.

Reply:
My (strong) recommendation is to create a vec, reserve space, push items into it, and call `into_boxed_slice` when you're done.